packages/order-source-api/src/app/metadata/order-source-specification.ts

```diff
@@ -1,18 +1,40 @@
 import { AccountConnectionSpecification } from "./account-connection-specification";
 
+/** @description This represents a single branded order source within our system */
 export interface OrderSourceSpecification {
+  /** @description This is a unique GUID identifier for this order source */
   Id: string;
+  /** @description The branded name for this order source @example "Shopify", "eBay", "Amazon" */
   Name: string;
+  /** @description Indicates whether or not our system should send emails with updates */
   SendEmail: boolean;
+  /** @description Indicates whether or not this marketplace allows custom mappings */
   HasCustomMappings: boolean;
+  /** @description Indicates whether or not orders can be refreshed */
   CanRefresh: boolean;
+  /** @description Indicates whether or not the order source allows users to leave feedback */
```

Comment:
```suggestion
  /** @description Indicates whether or not the order source allows sellers to leave feedback on customers */
```
At some point we should start marking fields like this as `deprecated`; I don't think anyone does this anymore.